.github/workflows/pytest-workflow.yml

```diff
@@ -122,7 +122,8 @@ jobs:
             tags: subworkflows/bcl_demultiplex
           - profile: "conda"
             tags: subworkflows/vcf_annotate_ensemblvep
-
+          - profile: "conda"
+            tags: subworkflows/fasta_clean_fcs
```

Comment:
I think this shouldn't be in the PR.

Reply:
It is mentioned just above by @sateeshperi to not change it.